### DISCUSSION TITLE
esp_adc_cal not found when calibrating.

### DISCUSSION THREAD

**Discussion by Ajb2k3** [micropython/micropython#10815] [ESP32]
I'm trying to compile a clean version of micropython for the ESP32-C3 using esp-idf 5xx and I keep running into the 
```
CMake Error at /Users/adambryant/esp-idf/tools/cmake/build.cmake:519 (message):
  ERROR: Cannot find versions of "espressif/esp_adc_cal" with version
  satisfying "*" for the current target "esp32"
```

How can I fix this as I can't find anything using google?

**Comment by Ajb2k3** [comment]
Ok so the issue is that esp_adc_cal has been replaced with esp_adc but how to make the compile use the new files.
sorry I don't know anything about cake or esp-idf.

**Comment by PBrunot** [comment]
It seems to me that the problem is that the version you're using, ESP-IDF 5.x, is not supported yet by Micropython. 
There are many breaking changes from 4.4 to it is not straightforward to upgrade (be patient...).
You can switch ESP-IDF to 4.4 to build a newer ESP32-C3 image.
